core/src/test/java/com/avast/dapper/ClusterBuilder.java

```diff
@@ -0,0 +1,360 @@
+package com.avast.dapper;
+
+import com.datastax.driver.core.*;
+import com.datastax.driver.core.policies.*;
+import com.google.common.base.Strings;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigException;
+import com.typesafe.config.ConfigFactory;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.SslProvider;
+
+import javax.net.ssl.*;
+import java.io.*;
+import java.net.InetSocketAddress;
+import java.security.GeneralSecurityException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class ClusterBuilder {
```

Comment:
Why copying this?

Reply:
Because I cannot use dependency on utils-datastax - it's not published...

Reply:
I know but maybe you could live without this class in tests?